recipes/sdl_mixer/cmake/conanfile.py

```diff
@@ -0,0 +1,207 @@
+        tc.variables["SDL2MIXER_VENDORED"] = False
+        tc.variables["SDL2MIXER_SAMPLES"] = False
+        tc.variables["SDL2MIXER_CMD"] = self.options.cmd
+        # WAVE
+        tc.variables["SDL2MIXER_WAVE"] = self.options.wav
+        # FLAC
+        tc.variables["SDL2MIXER_FLAC"] = self.options.flac
+        # MOD
+        tc.variables["SDL2MIXER_MOD"] = self.options.modplug
+        # MP3
+        tc.variables["SDL2MIXER_MP3"] = self.options.mpg123 or self.options.drmp3
+        tc.variables["SDL2MIXER_MP3_MPG123"] = self.options.mpg123
+        tc.variables["SDL2MIXER_MP3_DRMP3"] = self.options.drmp3
+        # MIDI
+        tc.variables["SDL2MIXER_MIDI"] = self.options.get_safe("nativemidi", False) or self.options.get_safe("tinymidi", False) or self.options.fluidsynth
+        tc.variables["SDL2MIXER_MIDI_FLUIDSYNTH"] = self.options.fluidsynth
+        tc.variables["SDL2MIXER_MIDI_TIMIDITY"] = self.options.get_safe("tinymidi", False)
+        tc.variables["SDL2MIXER_MIDI_NATIVE"] = self.options.get_safe("nativemidi", False)
+        # OPUS
+        tc.variables["SDL2MIXER_OPUS"] = self.options.opus
+        # VORBIS
+        if self.options.vorbis == "stb":
+            tc.variables["SDL2MIXER_VORBIS"] = "STB"
+        elif self.options.vorbis == "vorbisfile":
+            tc.variables["SDL2MIXER_VORBIS"] = "VORBISFILE"
+        elif self.options.vorbis == "tremor":
+            tc.variables["SDL2MIXER_VORBIS"] = "TREMOR"
+        else:
+            tc.variables["SDL2MIXER_VORBIS"] = False
+
+        # TODO: add support for dynamic loading of dependencies
+        tc.variables["SDL2MIXER_DEPS_SHARED"] = False
+        tc.variables["SDL2MIXER_FLAC_LIBFLAC_SHARED"] = False
+        tc.variables["SDL2MIXER_MIDI_FLUIDSYNTH_SHARED"] = False
+        tc.variables["SDL2MIXER_MIDI_TIMIDITY_SHARED"] = False
+        tc.variables["SDL2MIXER_MOD_MODPLUG_SHARED"] = False
+        tc.variables["SDL2MIXER_MOD_XMP_SHARED"] = False
+        tc.variables["SDL2MIXER_MP3_MPG123_SHARED"] = False
+        tc.variables["SDL2MIXER_OGG_SHARED"] = False
+        tc.variables["SDL2MIXER_OPUS_SHARED"] = False
+        tc.variables["SDL2MIXER_SNDFILE_SHARED"] = False
+        tc.variables["SDL2MIXER_VORBIS_TREMOR_SHARED"] = False
+        tc.variables["SDL2MIXER_VORBIS_VORBISFILE_SHARED"] = False
+        tc.generate()
+
+        deps = CMakeDeps(self)
+        deps.set_property("flac", "cmake_file_name", "FLAC")
+        deps.set_property("flac", "cmake_target_name", "FLAC")
+        deps.set_property("fluidsynth", "cmake_file_name", "FluidSynth")
+        deps.set_property("fluidsynth", "cmake_target_name", "FluidSynth::FluidSynth")
+        deps.set_property("libxmp", "cmake_file_name", "libxmp")
+        deps.set_property("libxmp", "cmake_target_name", "libxmp::libxmp")
+        deps.set_property("modplug", "cmake_file_name", "modplug")
+        deps.set_property("modplug", "cmake_target_name", "modplug::modplug")
```

Comment:
```suggestion
        deps.set_property("libmodplug", "cmake_file_name", "modplug")
        deps.set_property("libmodplug", "cmake_target_name", "modplug::modplug")
```

Here is your issue, https://github.com/conan-io/conan-center-index/pull/22170 is irrelevant.

Reply:
Thanks!